assets/css/common-styles.css

```diff
@@ -17,9 +17,11 @@ body {
 .text-center {
     text-align: center;
     align-items: center;
+    margin-bottom: 2px !important;
```

Comment:
Please don't use `!important`. This will impact other pages as well, unless you change all other pages accordingly. Otherwise, please define another class that is used specifically for index page (and then you can do something like `class="text-center your-index-page-class"`).